docs/pages/index.rst

```diff
@@ -5,8 +5,18 @@ A flexible, high-performance 3D simulator with configurable agents, multiple
 sensors, and generic 3D dataset handling (with built-in support for
 MatterPort3D, Gibson, Replica, and other datasets).
 
-When rendering a scene from the Matterport3D dataset, *Habitat-Sim* achieves :abbr:`several <really?>` thousand frames per second (FPS) running
-single-threaded, and reaches over 10,000 FPS multi-process on a single GPU!
+When rendering a scene from the Matterport3D dataset, *Habitat-Sim* achieves
+several thousand frames per second (FPS) running single-threaded, and reaches
+over 10,000 FPS multi-process on a single GPU!
+
+`Tutorials`_
+============
+
+.. TODO: this is waiting on m.css to propagate page titles to links
+
+-   `New Actions <std:doc:new-actions>`
+-   `Notebooks <std:doc:notebooks>`
+-   `Creating a stereo agent <std:doc:stereo-agent>`
```

Comment:
Should this be links yet?

Reply:
If you mean the TODO, no, that still holds ... i had to prioritize to not spend next 16 years on the generator :)

Reply:
Yep, no problem, just wanted to make sure there wasn't something I was missing :)